File: vaidik-server-main/list_models.js
const { OpenAI } = require('openai');

async function listModels() {
  const openai = new OpenAI({
    apiKey: "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"
  });

  try {
    const list = await openai.models.list();
    const sorted = list.data.map(m => m.id).sort();
    console.log(JSON.stringify(sorted, null, 2));
  } catch (error) {
    console.error('Error fetching models:', error.message);
  }
}

listModels();

File: vaidik-server-main/list_models.py
import os
from openai import OpenAI

client = OpenAI(api_key="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")

try:
    models = client.models.list()
    model_ids = [model.id for model in models.data]
    model_ids.sort()
    for m_id in model_ids:
        print(m_id)
except Exception as e:
    print(f"Error: {e}")
